concent_api/core/exceptions.py

```diff
@@ -23,3 +23,7 @@ class GolemMessageValidationError(ConcentBaseException):
 
 class FrameNumberValidationError(ConcentBaseException):
     pass
+
+
+class MessageValidationError(ConcentBaseException):
```

Comment:
How about reusing `ValidationError` from Django ?

Reply:
Primarily, `ValidationError` from Django is for forms and models' fields validation. We might use it but we would need to:
1. handle it in `handle_errors_and_responses` decorator
2. make sure, that catching it in the above mentioned decorator would not have any side-effects (i.e. catching model validation errors and returning it as bad request)

All in all, I would be for separate exception, specific to client's message validation error.